core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -238,7 +238,8 @@ angular.module('oppia').factory('ThreadDataService', [
           text: newMessage
         }).then((response) => {
           if (updatedStatus) {
-            openThreadsCount += (newStatus === STATUS_OPEN) ? 1 : -1;
+            openThreadsCount += (newStatus === STATUS_OPEN) ? 1 :
```

Comment:
I'm not sure this is the right thing to do. It is just showing the number of open threads, so it should be 
```suggestion
            openThreadsCount += (newStatus === STATUS_OPEN) ? 1 : 0 
```
right?

Reply:
@nithusha21 if we put 0(zero) there then it doesn't  updates the feed for fixed and others. And we can also see that if we put 0 there then count doesn't decreases for fixed but the status is updated as fixed and when we open it again its count increases again.

Reply:
we can also do something like this ((newStatus === STATUS_FIXED) ? -1 : 0) instead of 0(zero) in this it will only decrease the count for fixed suggestions and leave the count as it is for other status like ignored, complement and not actionable.